proposals/lip-0069.md

```diff
@@ -81,6 +81,10 @@ _Events_ are on-chain data emitted during the processing of a block which add ex
 
 Lisk SDK modules can define block generation hooks, state machine hooks and commands to add logic to the state machine. Each hook of the module will be called as described in the life cycle below (also described in [LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md)).
 
+
+For the hooks which call all registered modules, the hook of a module will be called in the order of the registration.
+In some modules, it requires methods from other modules, and for that, the dependency needs to be resolved before registering the module to the application.
```

Comment:
What do we mean concretely by "the dependency needs to be resolved"?

Reply:
What i wanted to mention was [these logics](https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/application.ts#L183-L200), where dependencies are manually resolved

Reply:
Then I would maybe explicitly say that the dependency must be 'stated' or 'added'?